runtime/caml/intext.h

```diff
@@ -109,8 +109,7 @@ void caml_output_val (struct channel * chan, value v, value flags);
   /* Output [v] with flags [flags] on the channel [chan]. */
 
 struct caml_intern_state;
-struct caml_intern_state* caml_alloc_intern_state (void);
-void caml_free_intern_state (struct caml_intern_state*);
+void caml_free_intern_state ();
```

Comment:
Please declare as
```
void caml_free_intern_state (void);
```
so that the C compiler can shout at you if you call it with arguments...